MH4U Dex/DamageZonesCollectionViewController.m

```diff
@@ -36,45 +31,50 @@ @implementation DamageZonesCollectionViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    self.flowLayout.itemSize = CGSizeMake(CGRectGetWidth(self.collectionView.bounds), 40.0f);
+
     self.damageZones = [self.monster.damageZone sortedArrayUsingDescriptors:@[[NSSortDescriptor
                                                                                sortDescriptorWithKey:DamageZoneAttributes.id
                                                                                ascending:YES]]];
 }
 
+- (void)viewWillLayoutSubviews
+{
+    CGFloat cellSize = CGRectGetWidth(self.collectionView.bounds);
```

Comment:
The documentation said viewWillLayoutSubviews doesn't do anything by default, so I didn't really think it was necessary.  I could add it, though.


The cellSize wiggling makes it so there are exactly 5 cells on every row, and that their sizes change dynamically based on the screen width.  The position and layout of the cells is deliberate, so there always need to be 5 cells per row.

I could probably have turned this into a tableView instead, but to be honest, I wanted to see how I would go about doing it in a collectionView.